.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build:
+  build-main:
```

Comment:
I wonder if this change caused it to wait for these to run?

<img width="965" alt="Screenshot 2023-08-30 at 5 01 46 PM" src="https://github.com/DataJunction/dj/assets/9524628/a6ce586b-4434-4ff5-b4be-485158793169">

Reply:
thank you, I will put that back